README.md

```diff
@@ -64,6 +64,27 @@ module.exports = {
 };
 ```
 
+If you are using the draft/publish feature, here is how to do it to no index unpublished content:
```

Comment:
Or maybe this is better:
```suggestion
If you are using the draft/publish feature, here is an example on how to only index published entries:
```

Reply:
Hey @francois2metz,

After rethinking it, I think _'If you are using the draft/publish feature, here is an example on how to only index published entries:'_ is better, could edit it into this?

Reply:
I did it.